plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_aws.yaml

```
@@ -0,0 +1,159 @@
+hydra:
+  launcher:
+    cls: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
+    params:
+      stop_cluster: True
+      # ray.init(**ray_init_config)
```

I don't like these comments, they look like commented out code.
```suggestion
      # parameters for ray.init()
```